docs/_posts/examples/3400-01-10-live-update-feature.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: example
+category: example
+title: Update a feature in realtime
+description: Change an existing feature on your map in realtime by updating its data
+tags:
+  - sources
+---
+<div id='map'></div>
+
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/satellite-v9',
+    zoom: 0
+});
+
+var url = '{{site.baseurl}}/assets/hike.geojson';
+map.on('load', function () {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
```

Comment:
In [another example](https://github.com/mapbox/mapbox-gl-js/blob/675e20c4ebe848401e7320d591f58f30b60037df/docs/_posts/examples/3400-01-20-timeline-animation.html#L112-L112) we use d3 to make the AJAX request -- I don't have a strong preference vs raw XHR, but we should have a consistent convention one way or the other.